core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -195,6 +198,21 @@ protected[controller] class RestAPIVersion_v1()(
     private val webexp = new WebActionsApi(Seq("experimental", "web"), WebApiDirectives.exp)
     private val web = new WebActionsApi(Seq("web"), WebApiDirectives.web)
 
+    /**
+     * Describes details of a particular API path.
+     */
+    private val info = JsObject(
+        "description" -> "OpenWhisk API".toJson,
+        "min_cli_version" -> whiskConfig(whiskVersionDate).toJson,
+        "min_recommended_cli_version" -> whiskConfig(whiskVersionDate).toJson,
```

Comment:
Yes but we have to version the cli first. I could take these out but since we keep getting bit by the cli out of date issues, this gives us a way to now have the cli check when it's out of date.